### PR TITLE
Updated user data to use api

### PR DESCRIPTION
**Pull request by SamphireReeve**

### CODE REVIEW

File: coalesce/users/models.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 class User(AbstractUser):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    first_name = models.TextField()
+    last_name = models.TextField()
```
Comment (stefanha) [inline coalesce/users/models.py:13]: Please run `docker-compose exec api ./manage.py makemigrations` and include the auto-generated migration file in this pull request. That will ensure that existing database tables are updated when this code lands.

Comment (SamphireReeve) [reply coalesce/users/models.py:13]: Ok will do

**Comment by stefanha** [comment]
There is one `docker-compose exec api ./manage.py test` failure. Looks good otherwise!

```
FAIL: coalesce.users.test.test_views:TestUserDetailTestCase.test_put_request_updates_a_user
  vi +54 coalesce/users/test/test_views.py  # test_put_request_updates_a_user
    eq_(response.status_code, status.HTTP_200_OK)
AssertionError: 400 != 200
```

File: coalesce/volunteers/models.py [hunk 2]
```diff
@@ -9,7 +9,7 @@ class Volunteer(models.Model):
         primary_key=True
     )
     description = models.TextField()
-    # TODO List of Opportunities they have indicated interest in
+    opportunities = [1]
```
Comment (stefanha) [inline coalesce/volunteers/models.py:12]: Was this line for testing? I don't think it's used.

**Pull request merged**